src/style.rei

```diff
@@ -23,292 +42,255 @@ let concat: list(t) => t;
  * Layout Props
  */
 let alignContent:
-  [ | `flexStart | `flexEnd | `center | `stretch | `spaceAround | `spaceBetween] => style;
-
-let alignItems: [ | `flexStart | `flexEnd | `center | `stretch | `baseline] => style;
-
-let alignSelf: [ | `flexStart | `flexEnd | `center | `stretch | `baseline] => style;
+  [ | `FlexStart | `FlexEnd | `Center | `Stretch | `SpaceAround | `SpaceBetween] =>
+  styleElement;
 
-let aspectRatio: float => style;
+let alignItems:
+  [ | `FlexStart | `FlexEnd | `Center | `Stretch | `Baseline] => styleElement;
```

Comment:
Why are we Capitalizing the variants? Doesn't it adhere to react-native stylesheets to have this camelCased?

Reply:
Because it's idiomatic for variants to be uppercased. I treat lower cased ones as legacy. Just for consistency sake with other codebases.

Reply:
But what about consistency with React-native styling and css. I think it would  be in sync with ReasonML's approach to be more attractive to developers coming from JS to keep the values similar to css.

Reply:
it's `flex-end`/`flex-start` anyway. So I'd even say `` `FlexStart `` is better because it literally removes the hyphen and capitalises.